main/guides/wallet/ui.md

```diff
@@ -3,127 +3,59 @@
 
 From a shell window, run `agoric open` to open the Wallet UI in a browser tab.
 
-## Menu Bar
+The wallet facilitates several core user interactions within the Agoric ecosystem, described in the below sections.
 
-At the top of the UI is a menu bar with four items.
+## <a name="dapps">Dapps</a>
 
-![Menu bar](./assets/0-MenuBar.png)
+To interact with a dapp, it first must be connected in the wallet. The dapp will attempt to connect to the wallet instance registered at https://local.agoric.com/.
 
-- **Inbox**
-  - ![Inbox](./assets/2-InboxWalletUI.png)
-  - Shows your offers, impending payments, Dapps, and purses.
-  - Lets you send payments, enable/disable Dapps and change their petname, approve/decline offers, or deposit impending payments.
-- **Transfers**
-  - ![Transfers](./assets/3-TransfersWalletUI.png)
-  - Shows your purses and contacts.
-  - Lets you send payments, import contacts by Board ID and give them a petname.
-- **Setup**
-  - ![Setup](./assets/4-SetupWalletUI.png)
-  - Shows your Dapps, issuers, and contacts. 
-  - Lets you create empty purses, import contacts by Board ID and give them a petname, and enable/disable Dapps and change their petname.
-- **Connected/Disconnect**
-  - ![Connected](./assets/5-ConnectWalletUI.png)
-  - Shows if the Wallet UI is connected to your ag-solo.
-  - Lets you connect the Wallet to or disconnect the Wallet UI from your ag-solo.
+Before the dapp is able to make offers, the connection must be accepted from the wallet. Incoming dapp connections can be managed from the Dashboard view:
 
-As there are only six page components, several of which are repeated on the three pages making up the Wallet UI,
-we will cover the components in detail rather than the pages.
+<img width="500" alt="Screen Shot 2021-12-01 at 7 30 32 PM" src="https://user-images.githubusercontent.com/8848650/144352840-06d3a7f4-d503-4d46-9a6e-dc65aab020ba.png">
```

Comment:
Please make meaningful `alt` attributes, both here and below.  It would also be worthwhile to put the screenshots in the `assets` folder to avoid taking a dependency on Github.

Reply:
Fixed